core/src/main/resources/hudson/slaves/EnvironmentVariablesNodeProperty/config.jelly

```diff
@@ -24,13 +24,13 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <f:entry title="${%List of key-value pairs}" help="${descriptor.getHelpPage()}">
+  <f:entry title="${%List of variables}" help="${descriptor.getHelpPage()}">
```

Comment:
Is renaming these a great idea rather than fixing it in the `properties` file? This breaks all translations.